opencensus/trace/ext/django/middleware.py

```diff
@@ -85,11 +86,28 @@ def get_django_header():
     :rtype: str
     :returns: Trace context header in HTTP request headers.
     """
+    headers = get_django_headers([_DJANGO_TRACE_HEADER])
```

Comment:
If we want to add `get_django_headers(header_names)`, we could remove the `get_django_header()` method, since they basically have no difference. And it would be easy to make a typo calling the wrong method.